terraform/variables.tf

```diff
@@ -145,4 +145,8 @@ variable "inspec_db_migration_service" {
 
 variable "inspec_container_group_name" {
   default = "inspec_container_trial"
+}
+
+variable "sample_directory_object" {
+  default = "adc07321-ef2b-44d5-a210-559aa5f10f2d"
```

Comment:
Do we have to specify the hardcoded value? In AWS we specify the hardcoded values in test/integration/configuration/aws_inspec_config.rb